mindmeld/components/entity_recognizer.py

```diff
@@ -79,6 +79,7 @@ def fit(self, queries=None, label_set=None, incremental_timestamp=None, **kwargs
         logger.info('Fitting entity recognizer: domain=%r, intent=%r', self.domain, self.intent)
 
         # create model with given params
+        kwargs['tokenizer'] = self._resource_loader.query_factory.tokenizer
         self._model_config = self._get_model_config(**kwargs)
         model = create_model(self._model_config)
```

Comment:
I propose that we could pass the tokenizer here instead of putting it in the config:
```suggestion
        model = create_model(self._model_config, tokenizer=self._resource_loader.query_factory.tokenizer)
```

Reply:
resolved